src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -28,19 +28,21 @@
 using System.Linq;
 using System.Net;
 using Grpc.Core;
-using Microsoft.VisualStudio.Shell;
 using Sonarlint;
 
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
     [Export(typeof(ISonarLintDaemon))]
-    class SonarLintDaemon : ISonarLintDaemon
+    internal class SonarLintDaemon : ISonarLintDaemon
```

Comment:
Doesn't need to be publicly visible - MEF can still create it, and the tests can see the internals too.